data/language/en-GB.txt

```diff
@@ -3680,6 +3680,9 @@ STR_6574    :Void
 STR_6575    :Allow special colour schemes
 STR_6576    :Adds special colours to colour dropdown
 
+STR_6577    :Reset
```

Comment:
Reset Shortcuts

Reply:
This is the text on the prompt button. I think just `Reset` is better here, it also corresponds with something like `Demolish` on the prompt before you delete a ride.

Reply:
okay.